Felienne/hedy
# [UI IMPROVEMENT] Improves styling of read aloud dropdown and code repair lighbulb

**Description**
In this PR we fix the styling of the read aloud dropdown menu to ensure it looks the same on each browser. We also fix the styling of the code repair lightbulb as the current styling was a bit "houtje-touwtje". See the screenshots below for clarification.

**Fixes**
This PR fixes #2221

**How to test**
Navigate to the code page and notice the new styling of the read aloud dropdown menu. Try to run a faulty program and notice the new location of the lightbulb, better aligning with the rest of our UI.

**Old view**
![Schermafbeelding 2022-03-15 om 16 07 40](https://user-images.githubusercontent.com/38155424/158408944-441a53ae-92fd-4bfc-a88d-b9696049787e.png)

**New view**
![Schermafbeelding 2022-03-15 om 16 07 06](https://user-images.githubusercontent.com/38155424/158408961-69dc0aa3-5dc1-4e6b-be82-3480d7bf589d.png)
